src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -105,12 +105,32 @@ internal sealed class LogLossFactory : ISupportSdcaClassificationLossFactory, IS
         IClassificationLoss IComponentFactory<IClassificationLoss>.CreateComponent(IHostEnvironment env) => new LogLoss();
     }
 
+    /// <summary>
+    /// The Log Loss, also known as the Cross Entropy Loss. It is commonly used in classification tasks.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// The Log Loss function is defined as:
+    ///
+    /// $L(f(\vec{x}, y) = -y ln(f(\vec{x})) - (1 - y) ln(1 - f(\vec{x}))$
+    ///
+    /// where $f(\vec{x})$ is the predicted probability of belonging to the positive class and $y \in \{0, 1\}$ is the true label.
+    ///
+    /// Note that the labels used in this calculation are 0 and 1, unlike Hinge Loss and Exponential Loss, where the labels used are -1 and 1.
+    ///
+    /// The Log Loss function provides a measure of how "certain" a classifier's predictions are, instead of just measuring how "correct" they are.
```

Comment:
>Log [](start = 12, length = 3)

you have kink on wikipedia hinge loss, any reason why you decide not add link for log loss?

Reply:
Hinge loss had some interesting details regarding differentiability, that's why I added the link. Log loss section on that article doesn't have much to add.